### PR TITLE
chore: stop using `null` for settings

### PR DESCRIPTION
**Pull request by Malix-off**
Best practice

### CODE REVIEW

**Comment by nhedger** [comment]
Anywhere this best practice is documented? Not sure what the issue is here.

**Comment by Malix-off** [comment]
@nhedger

This is due to the UI change in the settings:

## Diff Preview

### Before

image

### After

image

**Comment by nhedger** [comment]
That makes sense; thank you!

File: package.json [hunk 1]
```diff
@@ -84,13 +84,11 @@
...
 				"biome.lspBin": {
-					"type": ["string", "null"],
-					"default": null,
+					"type": "string",
 					"markdownDescription": "The biome lsp server executable. If the path is relative, the workspace folder will be used as base path"
 				},
 				"biome.rename": {
-					"type": ["boolean", "null"],
-					"default": null,
+					"type": "boolean",
```
Comment (nhedger) [inline package.json:91]:
```suggestion
					"type": "boolean",
					"default": false,
```

File: package.json [hunk 2]
```diff
@@ -84,13 +84,11 @@
 					"description": "Traces the communication between VS Code and the language server."
 				},
 				"biome.lspBin": {
-					"type": ["string", "null"],
-					"default": null,
+					"type": "string",
```
Comment (nhedger) [inline package.json:87]:
```suggestion
					"type": "string",
					"default": "",
```

File: package.json (hunk 1 above)
Comment (Malix-off) [reply package.json:91]: Similar behavior, best if you want declarative, worst if you want less LOC and default

File: package.json (hunk 2 above)
Comment (Malix-off) [reply package.json:87]: > Similar behavior, best if you want declarative, worst if you want less LOC and default 👍

Comment (nhedger) [reply package.json:87]: I'd rather be explicit here.

File: package.json (hunk 1 above)
Comment (nhedger) [reply package.json:91]: I'd rather be explicit here.

File: package.json [hunk 3]
```diff
@@ -84,6 +84,7 @@
 				"biome.lspBin": {
 					"type": "string",
 					"default": "",
+          "ignoreSync": true,
```
Comment (Malix-off) [inline package.json:87]:
```suggestion
					"ignoreSync": true,
```

**Comment by nhedger** [comment]
Thanks!

**Pull request merged**